README.md

```diff
@@ -412,6 +419,70 @@ Fireballs
   .catch(err => console.log(err));
 ```
 
+<h3 id='images-search'>Images.search(options)</h3>
+Retrieves results and information on how to retrieve more details on images taken by astronauts
+
```

Comment:
add a link to API docs

For additional information, see the [API Documentation](https://images.nasa.gov/docs/images.nasa.gov_api_docs.pdf)